### DISCUSSION TITLE
Lab 4, Part 5 question on screenshot

### DISCUSSION THREAD

**Discussion by joempappas** [ag-informatics/ag-informatics-course#24] [Troubleshooting]
Part 5 asks us to include a screenshot:
```
IMAGE: Take a screenshot a single filed that contains several data entires. This should correspond to the second view you created. Save it with the filename 'sample-view' inside our 'lab4/images' folder.
```

I don't really understand whst this is asking. I assume the 'filed' is supposed to be 'field', but even still, could someone help clear up what this is asking?

**Comment by tame0001** [comment]
The first view should be a view of fields (same as previous lab). Then when you click on one of them, it should redirect to a second view that shows details of that particular field. The content of this view will be an operation in chronological order
